apps/e2e-tests/tests/tree-widget.spec.ts

```diff
@@ -0,0 +1,76 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { test, expect } from '@playwright/test';
+import assert from "assert";
+import { expandStagePanel, locateNode, locateWidget } from "./utils";
+
+test.describe("tree-widget", () => {
+
+  test.beforeEach(async ({ page, baseURL }) => {
+    assert(baseURL);
+    await page.goto(baseURL);
+  })
+
+  test.describe("should match image snapshot", () => {
+
+    const testCases = () => {
+
+      test("initial tree", async ({ page }) => {
+        const treeWidget = locateWidget(page, "tree");
```

Comment:
this is repeated in all tests - maybe it should be moved to `beforeEach`?

Reply:
But I will still need to locate this widget to get a screenshot of it.

Reply:
maybe do that like this?
```
let widget;
beforeEach(() => {
  widget = locateWidget(...);
});
test(() => {
  // do something with widget
});
```

Reply:
I have fixed it in `TreeWidget` test. Tried to do it in `PropertyWidget` test, looks a bit messy.